test/mocha.opts

```diff
@@ -1,4 +1,5 @@
 --require babel-register
+--require jsdom-global/register
```

Comment:
We'll either need to, or instruct the user to shim for server side rendering. e.g. `var HTMLElement = typeof HTMLElement === 'undefined' ? function(){} : HTMLElement;`

[This solution found here](https://github.com/ceolter/ag-grid-react/issues/24#issuecomment-244678042)